arbitrator/stylus/src/cache.rs

```diff
@@ -77,6 +79,14 @@ impl InitCache {
             .resize(NonZeroUsize::new(size.try_into().unwrap()).unwrap())
     }
 
+    pub fn set_target(target: Target) {
+        cache!().target = target;
```

Comment:
This function was removed but it's replaced by calling target_native from the cache which could, theoretically, return a different target over time.
However, I don't think it's a problem, because even though target_native() could change, it will always be compatible with native - so it'll always be for the same cpu architecture and operating system. I don't believe deserialization/cache contents could become wrong when we replace native instance.